Justfile

```diff
@@ -67,6 +67,7 @@ init-db:
 # Regenerate SQLC code (requires init-db to be run first)
 build-sqlc:
   @mk backend/controller/sql/{db.go,models.go,querier.go,queries.sql.go} : backend/controller/sql/queries.sql backend/controller/sql/schema sqlc.yaml -- "just init-db && sqlc generate"
+  @mk common/configuration/sql/{db.go,models.go,querier.go,queries.sql.go} : common/configuration/sql/queries.sql backend/controller/sql/schema sqlc.yaml -- "just init-db && sqlc generate"
```

Comment:
Combine these - mk can take any number of inputs and any number of outputs

Reply:
sweet! done